resources/assets/js/components/partials/file.vue

```diff
@@ -64,6 +69,21 @@ export default {
           this.showNotification({type: error.response.data.status, message: error.response.data.message})
         })
     },
+    downloadFile () {
+      axios.get(`https://localhost/storage/${this.file.name}`, {responseType: 'blob'})
```

Comment:
don't hard code the url here, just use relative path

Reply:
I have updated the relative path. Please check.